projects/ng-animate/package.json

```diff
@@ -17,11 +17,11 @@
   },
   "homepage": "https://github.com/jiayihu/ng-animate#readme",
   "peerDependencies": {
-    "@angular/common": ">=9.0.0",
-    "@angular/core": ">=9.0.0",
-    "@angular/animations": ">=9.0.0"
+    "@angular/common": ">=14.0.0",
```

Comment:
According to the doc, partial Ivy compilation is supported by Angular 12+ so I think we should support more versions if possible